.env

```diff
@@ -10,6 +10,9 @@ REACT_APP_SHOW_EMAIL_LOGIN=false
 REACT_APP_SHOW_OIDC_LOGIN=true
 REACT_APP_SHOW_OIDC_LOGIN_GOOGLE=true
 
-# These settings control how much logging is done for SWR
+# Control how much logging is done for SWR
 REACT_APP_SWR_LOG_CACHE=false
-REACT_APP_SWR_LOG_REQUESTS=false
\ No newline at end of file
+REACT_APP_SWR_LOG_REQUESTS=false
+
+# This is the title displayed while the UI is initially loading in the browser
+REACT_APP_DEFAULT_PAGE_TITLE=Estuary Flow
```

Comment:
Eventually we should be smart and pull a variable from the lang pack.... but this needs to be "hardcoded" into the index.html for right now